components/Form.js

```diff
@@ -13,15 +12,51 @@ const StyledForm = styled.form`
   gap: 0.5rem;
 `;
 
-export default function Form({ onSubmit, entrySucessful }) {
-  const router = useRouter();
+export default function Form({ onSubmit, entrySuccessful }) {
+  const [checkedSeasons, setCheckedSeasons] = useState({
+    Spring: false,
+    Summer: false,
+    Fall: false,
+  });
+  function handleCheckboxChange(event) {
+    const { name, checked } = event.target;
+    setCheckedSeasons((checkedSeason) => ({
+      ...checkedSeason,
+      [name]: checked,
+    }));
+  }
 
   function handleSubmit(event) {
     event.preventDefault();
     const formData = new FormData(event.target);
     const data = Object.fromEntries(formData);
+
+    //delete entries from data
+    delete data.Spring;
+    delete data.Summer;
+    delete data.Fall;
+
+    //input type checkbox saved as a array
+    //   const fertiliserSeason = [];
+    //   document
+    //     .querySelectorAll('input[type="checkbox"]:checked')
+    //     .forEach((checkbox) => {
+    //       fertiliserSeason.push(checkbox.value);
+    //     });
+
+    //   data.fertiliser_season = fertiliserSeason; //save checkbox array in object
+    //   onSubmit(data);
+    //   entrySucessful();
+    // }
```

Comment:
We should remove this comment before merging to main. If you want to find the code again you can always look through previous commits. i.e. [here](https://github.com/Fabi911/Capstone_Plant_Care/blob/4cac275395ecfa827403ca35e60440ce72d2c9ce/components/Form.js)